cmd/issuer-rest/static/storeprc.html

```diff
@@ -170,12 +192,13 @@ <h2 class="bold text-5xl px-2">Digital Permanent Resident Card</h2>
                     Permanent Resident Card
                 </label>
                 <label class="block text-sm font-sans text-gray-600 mb-2 py-4" for="email">
-                    You can click on Connect to wallet button to connect to wallet, or scan the QR Code to store the resident card.
+                    You can click on Connect to wallet button to connect to wallet, or scan the QR Code to store the
+                    resident card.
                 </label>
-                <a class="bg-blue-800 py-2 px-4 border border-gray-800 hover:bg-green-dark text-white font-bold rounded shadow-lg
+                <button class="bg-blue-800 py-2 px-4 border border-gray-800 hover:bg-green-dark text-white font-bold rounded shadow-lg
```

Comment:
Only this was the problem, anchor tag wasnt picking the color in safai, Rest all are linter spaces removed/added by ES linter